generic_relations/tests/models.py

```diff
@@ -1,5 +1,11 @@
 from django.contrib.contenttypes.models import ContentType
-from django.contrib.contenttypes import generic
+
+try:
+    from django.contrib.contenttypes.fields import GenericForeignKey, GenericRelation
+except ImportError:
+    # django < 1.7
+    from django.contrib.contenttypes.fields import GenericForeignKey, GenericRelation
```

Comment:
Ha, good catch. It was meant to be 
```
    from django.contrib.contenttypes.fields import GenericForeignKey, GenericRelation
```

However if we're explicitly dropping django 1.6 anyway there's no need for this